index.js

```diff
@@ -0,0 +1,14 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2018, Alert Logic, Inc
+ * @doc
+ *
+ * Classes for communication to Alert Logic services.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+module.exports = {
+    AlServiceC : require('./al_servicec')
```

Comment:
Shall I actually enumerate all classes we have in `al_servicec.js` here?

Like

```
module.exports = {
    AimsC : require('./al_servicec').AimsC,
    AlServiceC : require('./al_servicec').AlServiceC,
    IngestC : require('./al_servicec').IngestC,
    AzcollectC : require('./al_servicec').AzcollectC,
    EndpointsC : require('./al_servicec').EndpointsC,
    RestServiceClient : require('./al_util').RestServiceClient
};
```
?

Reply:
@kkuzmin here https://github.com/alertlogic/al-aws-collector-js/blob/f2a1d0b25acfd8c090a1b3b3e2e72e29fdbc8589/index.js#L12 you just reffered file, not the class btw, is that how it should be?

Reply:
Are there any references to `require('./al_util').RestServiceClient`? Do we want to explore it as well?

Reply:
If we do like this then consumer code would look like
```
require('al-collector-js'').AlServiceC.IngestC.
```

So it is probably worth doing like you;ve suggested in the comment
```
module.exports = {
    AimsC : require('./al_servicec').AimsC
}
```

Reply:
@kkuzmin I'll do this way  then (the one I suggested) 👍